### DISCUSSION TITLE
Theming & translating

### DISCUSSION THREAD

**Discussion by Valexr** [nuejs/nue#418] [Q&A]
I can't get browser lang or system theme in layout.html file, because object window/document not accessible...

How I can get browser lang or system theme in layout.html?

**Comment by tipiirai** [comment]
You can set it on site.yaml with `language` property. Use `direction` if you want to change the HTML dir property

**Comment by Valexr** [comment]
I don't need set lang, I need get it from client browser 🤓 for modernization layout.html

**Comment by tipiirai** [comment]
Not 100% sure what you try to accomplish, but seems you need a global javascript file to manipulate the DOM on the client side.

**Comment by Valexr** [comment]
Maybe… it’s just simple clientUX case - get browser lang for translate page for prefer user language.

I see island have client side js logic… but not useful in layouts 🤔 html
